BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -36,9 +36,9 @@ int update_Time(int row)
 
 }
 
-int writeReload(int reload)
+int8_t BSP_Timer_Init(int8_t reload)
```

Comment:
This function shouldn't return anything. Also, it should set some sort of static global variable in this file that is the function pointer to whatever function needs to be called on interrupt.